checkov/sca_package/scanner.py

```diff
@@ -53,6 +57,28 @@ async def run_scan_multi(
         else:
             scan_results = await asyncio.gather(*[self.run_scan(i) for i in input_paths])
 
+        if any(scan_result["vulnerabilities"] is None for scan_result in scan_results):
+            self.setup_twistcli()
+
+            if os.getenv("PYCHARM_HOSTED") == "1":
+                # PYCHARM_HOSTED env variable equals 1 when running via Pycharm.
+                # it avoids us from crashing, which happens when using multiprocessing via Pycharm's debug-mode
+                logging.warning("Running the scans in sequence for avoiding crashing when running via Pycharm")
+                scan_twist_results = []
+                for idx, input_path in enumerate(input_paths):
+                    if scan_results[idx]["vulnerabilities"] is None:
+                        scan_twist_results.append(await self.execute_twistcli_scan(input_path))
+            else:
+                scan_twist_results = await asyncio.gather(*[
+                    self.execute_twistcli_scan(input_path) for idx, input_path in enumerate(input_paths) if scan_results[idx]["vulnerabilities"] is None
+                ])
```

Comment:
How about this, and then no need for the re-fill?
```suggestion
                scan_twist_results = await asyncio.gather(*[
                    self.execute_twistcli_scan(input_path) if scan_results[idx]["vulnerabilities"] is None else scan_results[idx] for idx, input_path in enumerate(input_paths) 
                ])
```